MAINTAINERS.md

```diff
@@ -0,0 +1,65 @@
+# Maintainers
+
+This document contains list of present maintainers, their role and selection process.
+
+## Current Maintainers
+
+|   Username    |    Focus Area   |
+| ------------- | -------------   |
+|  MarcoFalke   | General, QA     | 
+|  fanquake     | General, Build  |
+|  hebasto      | General, UI/UX  |
+|  achow101     | General, Wallet |
+|  glozow       | General, Mempool|
+
+## Responsibilities and roles
+
+Project maintainers have commit access and are responsible for merging patches 
+from contributors. They perform a janitorial role merging patches that the team 
+agrees should be merged. They also act as a final check to ensure that patches 
```

Comment:
A maintainer decision isn't final. It is merely a piece before post-merge review and post-merge testing happens. Also, there is a release process and the every user is free to use their own judgement whether and what software they want to run.

Reply:
I copied this part from https://bitcoincore.org/en/about/. Rephrased 'final check' and added a sentence based on your suggestion.